pool/app/system_event/event.ml

```diff
@@ -0,0 +1,65 @@
+open CCFun
+open Entity
+
+type event = Created of t [@@deriving show, eq, variants]
+
+let handle_event : event -> unit Lwt.t = function
+  | Created t -> Repo.insert Pool_database.root t
+;;
+
+let handle_system_event system_event =
+  let open Utils.Lwt_result.Infix in
+  let pool = Pool_database.root in
+  let created_at = Pool_common.CreatedAt.create () in
+  let updated_at = Pool_common.UpdatedAt.create () in
+  let success identifier =
+    EventLog.
```

Comment:
I'd suggest using directly `EventLog.create system_event.id identifier Status.Successful`

[Link to implementation](https://github.com/uzh/pool/pull/145/files#diff-2f5d2972b5145816fcac7c8ce8f5d998efa04fedba5af1f31690cb05a823af01R106)